docs/source/docker/advanced/active_learning.rst

```diff
@@ -1,7 +1,294 @@
 .. _ref-docker-active-learning:
 
-Active learning
-==============================================
+Active Learning
+===============
+
+Lightly makes use of active learning scores to select the samples which will yield
+the biggest improvements of your machine learning model. The scores are calculated
+on-the-fly based on model predictions and provide the selection algorithm with feedback
+about the uncertainty of the model for the given sample. 
+
+Learn more about the concept of active learning scores:
+:ref:`lightly-active-learning-scorers`.
+
+Prerequisites
+--------------
+In order to do active learning with Lightly, you will need the following things:
+
+- The installed Lightly docker (see :ref:`ref-docker-setup`)
+- A dataset with a configured datasource (see :ref:`ref-docker-with-datasource-datapool`)
+
+.. note::
+
+    The dataset does not need to be empty! For example, an initial selection without
+    active learning can be used to train a model. The predictions from this model
+    can then be used to improve your dataset by adding new images to it through active learning.
+
+
+Next, you need to prepare the model predictions in the correct format.
+
+Predictions
+-----------
+
+In the following, we will outline the format of the predictions required by the 
+Lightly docker. Everything regarding predictions will take place in a subdirectory
+of your configured datasource called `.lightly/predictions`.
+
+Prediction Tasks
+^^^^^^^^^^^^^^^^
+To let Lightly know what kind of prediction tasks you want to work with, Lightly
+needs to know their names. It's very easy to let Lightly know which tasks exist:
+simply add a `tasks.json`` in your storage bucket stored at the subdirectory `.lightly/predictions/`.
+
+The `tasks.json` file must include a list of your task names which must match name
+of the subdirectory where your prediction schemas will be located.
+
+.. note::
+
+    Only the task names listed within `tasks.json`` will be considered.
+    Please ensure that the task name corresponds with the location of your prediction schema.
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/tasks.json
+
+    [
+        "my_classification_task",
+        "my_object_detection_task"
+        "${TASK_NAME}",
+    ]
+
+
+Prediction Schema
+^^^^^^^^^^^^^^^^^
+For Lightly it's required to store a prediction schema. For classification and object
+detection the prediction schema must include all the categories and ids. For other
+tasks such as keypoint detection it can be useful to store additional information
+like edges between keypoints. You can provide this information to Lightly by adding
+a `schema.json`` file to the directory of the respective task.
+
+The schema.json file must have a key categories with a corresponding list of categories following the COCO annotation format:
+
+
+.. code-block:: javascript
```

Comment:
Drop the whole code here and just show the "real" example with the weather classes. This block here does not help.